numpy/_core/src/umath/highway_transform.h

```diff
@@ -0,0 +1,75 @@
+#ifndef _NPY_CORE_SRC_UMATH_HIGHWAY_TRANSFORM_H_
+#define _NPY_CORE_SRC_UMATH_HIGHWAY_TRANSFORM_H_
+
+#include "simd/simd.h"
+#include <hwy/highway.h>
+#include <numpy/npy_common.h>
+
+#if NPY_SIMD
+HWY_BEFORE_NAMESPACE();
+namespace hwy {
+namespace HWY_NAMESPACE {
+
+// Sets `out[idx]` to `Func(d, in[idx])`
+// Useful for math routines.
+template <class D, class Func, typename T = TFromD<D>>
+HWY_ATTR void
+Transform1(D d, const T *HWY_RESTRICT in, const npy_intp in_stride,
```

Comment:
Probably rename this too?

Reply:
Any suggestions welcome 😸 Maybe something like `LoadStoreAndTransform`? Naming is hard.

Reply:
I suggest `LoadComputeStore`: perfectly describes what it is doing :)

Reply:
I've gone with `ComputeUnary`, the `Load` and `Store` aspects seem to be overly describing the details of the function.